df_py/util/dftool_module.py

```diff
@@ -595,7 +596,13 @@ def do_dispense_active():
         predictoor_rewards_3d = load_predictoor_rewards_csv(arguments.CSV_DIR)
         predictoor_rewards = calc_rewards.flatten_rewards(predictoor_rewards_3d)
 
-    rewards = calc_rewards.merge_rewards(volume_rewards, predictoor_rewards)
+    challenge_rewards = {}
```

Comment:
what happens if there is no rewards file? does it not dispense anything? Should the user be warned or something?